docs/getting_started.md

```diff
@@ -0,0 +1,363 @@
+
+### Actions
+The brand new APB created in the last section doesn't do very much.  For that, we'll have to add some actions.  The actions supported are [provision](#provision), [deprovision](#deprovision), [bind](#bind), and [unbind](#unbind).  We'll add each of these actions in the following sections.
+
+Before we begin, make sure you're logged in.
+```
+# substitute your information for <catasb-host>:<catasb-port>
+oc login --insecure-skip-tls-verify <catasb-host>:<catasb-port> -u admin -p admin
+```
+
+[TODO]: # (change the example yaml so that service/route/dc are all different names to explicitly show the relationships specified by selector, etc)
+
+#### Provision
+Using the initialized apb from the last section, we can build and run the provision functionality (substitute your own docker organization name and catasb host/port where appropriate):
+<a name="run-provision-cmd">
+```
+# build the apb image
+apb build <docker-org>/my-apb
+
+# run provision task
+docker run \
+    -e "OPENSHIFT_TARGET=https://<catasb-host>:<catasb-port>" \
+    -e "OPENSHIFT_USER=admin" \
+    -e "OPENSHIFT_PASS=admin" \
+    <docker-org>/my-apb provision \
+    --extra-vars 'namespace=my-apb'
+```
+The environment arguments passed in using the `-e OPENSHIFT_TARGET`, `-e OPENSHIFT_USER`, and `-e OPENSHIFT_PASS`, will be used for the apb container to log in to the OpenShift cluster and are required.  The apb image tag `<docker-org>/my-apb` and action `provision` specify what to do.  The `--extra-vars 'namespace=my-apb'` is used to supply variables to the Ansible playbooks we'll write.  Running the above will output `'provision' NOT IMPLEMENTED`.
+
+To add the provision action, we'll need to add a yaml file `provision.yml` inside the playbooks directory.
+```
+my-apb/
+├── apb.yml
+├── Dockerfile
+├── playbooks
+│   └── provision.yml
+└── roles
+```
+`provision.yml` is the Ansible playbook that will be run when the **_provision_** action is called from the Ansible Service Broker.  Paste in the following code:
+```yaml
+- name: provision my-apb
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  roles:
+  - role: ansible.kubernetes-modules
+    install_python_requirements: no
+  - role: provision-my-apb
+    playbook_debug: false
+```
+
+`provision.yml` is a new Ansible playbook which will execute on `localhost` and execute the role `provision-my-apb`.  This playbook works on it's local container created by the service broker.  The `ansible.kubernetes-modules` role will allow us to use the [ansible-kubernetes-modules](https://github.com/ansible/ansible-kubernetes-modules) to create our OpenShift resources.  The `provision-my-apb` doesn't exist yet, but that's where all the work will go when deploying our hello-world app.  Let's create it now.
+
+First, create a new file called main.yml inside a roles/provision-my-apb directory.
```

Comment:
add tick marks around yml and directory:
`main.yml` and `roles/provision-my-apb`.